src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -491,33 +491,46 @@ public static void TCP_ServiceCertExpired_Throw_MessageSecurityException()
 
     [Fact]
     [OuterLoop]
+    // Confirm that the Validate method of the custom X509CertificateValidator is called and that an exception thrown there is handled correctly.
     public static void TCP_ServiceCertFailedCustomValidate_Throw_Exception()
     {
         string testString = "Hello";
+        NetTcpBinding binding = null;
+        EndpointAddress endpointAddress = null;
+        ChannelFactory<IWcfService> factory = null;
+        IWcfService serviceProxy = null;
```

Comment:
Yes, I tend to like the variable declaration in a non-throwable part of the code like @StephenBonikowsky is doing.  Then we can try/catch around the "work" part and still access them in the finally